design-document.md

```diff
@@ -12,6 +93,9 @@
 
 ## System design
 
-## Open questions and system parameters
+## Open questions and system parameters {#questions-and-params}
+
+[[OPEN ISSUE: discuss batch size parameter and thresholds]]
+[[OPEN ISSUE: discuss f^ leakage differences from HCG's paper]]
```

Comment:
What does "f^" mean?

Reply:
To resolve this, please add a reference to the paper.

Reply:
Can you please leave what you're looking for as a suggestion? (There is a reference -- "HCG's paper" -- no?)